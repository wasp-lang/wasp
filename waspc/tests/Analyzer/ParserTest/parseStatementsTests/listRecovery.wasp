app todoApp {
  dependencies: [("wasp-compiler"), ("wasp-cli")]
}

