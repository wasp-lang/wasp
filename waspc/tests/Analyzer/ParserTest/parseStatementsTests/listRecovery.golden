Parse error at 2:18-34:
  Tuple only contains 1 values, but it must contain at least 2 values

Parse error at 2:37-48:
  Tuple only contains 1 values, but it must contain at least 2 values
